packages/council-cli/src/commands/encode/treasury/approve.ts

```diff
@@ -72,6 +71,6 @@ export function encodeApprove(
   return encodeFunctionData({
     abi: Treasury__factory.abi,
     functionName: "approve",
-    args: [token, parseBigInt(amount, decimals), spender],
+    args: [token, parseUnits(amount as `${number}`, decimals), spender],
```

Comment:
Ran into this nasty casting as well trying to use parseUnits before! I liked the suggestion in this thread on creating a workaround fn until viem relaxes this type:

https://github.com/wagmi-dev/viem/discussions/490#discussioncomment-5904696

Reply:
Ah, yeah. would be useful